src/lib/terraform.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse} from 'axios'
+
+export interface TerraformOptions {
+  token: string,
+  host: string,
+}
+
+export {AxiosInstance as TerraformInstance}
+export {AxiosResponse as TerraformResponse}
+
+export function terraform ({ token, host = 'app.terraform.io' } : TerraformOptions): AxiosInstance {
+  return axios.create({
+    baseURL: `https://${host}`,
```

Comment:
`/api/v2/` should be included here. This is a client for that version of the API. If there were another version of the API used concurrently, it would have an entirely separate client.